src/NServiceBus.Core/Scheduling/Schedule.cs

```diff
@@ -3,36 +3,24 @@ namespace NServiceBus
     using System;
     using System.Linq;
     using System.Threading;
-    using ObjectBuilder;
+    using NServiceBus.Logging;
     using Scheduling;
 
     /// <summary>
-    /// Scheduling capability to schedule a task (as an <see cref="System.Action"/>) to be executed repeatedly in a given interval.
+    /// Extends the bus with scheduling capabilities.
     /// </summary>
-    /// <remarks>This is a in-memory list of <see cref="System.Action"/>s.</remarks>
-    public class Schedule
+    public static class ScheduleBusExtensions
```

Comment:
At first glance I don't see how these Scheduler changes are related to the bus startup? Are they necessary? What are they changing?